src/main/java/seedu/bigpp/pc/PC.java

```diff
@@ -30,11 +30,25 @@ public PC(String name, Boolean isPrebuilt) {
         this.isPrebuilt = isPrebuilt;
     }
 
+    public PC(PC pcCopy) {
+        this.name = pcCopy.getName() + " (copy)";
+        this.isPrebuilt = false;
+        this.cpu = pcCopy.getCpu();
+        this.cpuCooler = pcCopy.getCpuCooler();
+        this.gpu = pcCopy.getGpu();
+        this.motherboard = pcCopy.getMotherboard();
+        this.ram = pcCopy.getRam();
+        this.storage = pcCopy.getStorage();
+        this.psu = pcCopy.getPsu();
+        this.chassis = pcCopy.getChassis();
+        this.budget = pcCopy.getBudget();
+    }
+
```

Comment:
is overloading the constructor a good idea? or would it be better to create a method for creating a pc copy

Reply:
Think it is ok, if it grows into an issue we can change it out